conformance_tests/sysman/test_sysman_driver/README.md

```diff
@@ -0,0 +1,18 @@
+# testSysmanDriver
+
+## Description
+
+This test suite is for validating driver Extension APIs provided in Sysman. Ensure that the test executable is run with
+root user permissions otherwise test will fail.
```

Comment:
Why is this requirement .. Is it from spec ?

Reply:
Modified!